prime-router/build.gradle.kts

```diff
@@ -567,6 +567,7 @@ tasks.azureFunctionsRun {
 
     val env = mutableMapOf<String, Any>(
         "AzureWebJobsStorage" to devAzureConnectString,
+        "AzureBlobDownloadRetryCount" to 5,
```

Comment:
So I ran the smoketests with commented out and everything seemed to be fine (I assume because of the default in the code), so not strictly required, but seems fine to keep.  I would just maybe leave a comment here or in BlobAccess to keep these the same.